lib/normalizestyle.js

```diff
@@ -46,8 +53,14 @@ function normalizeFilled(properties) {
 
 function normalizePoint(properties) {
     if (!markersize.test(properties['marker-size'])) delete properties['marker-size'];
-    if (!hexcolor.test(properties['marker-color'])) delete properties['marker-color'];
-    if (properties['marker-color'] && properties['marker-color'][0] !== '#') properties['marker-color'] = '#' + properties['marker-color'];
+    if (typeof properties['marker-color'] !== 'undefined') {
```

Comment:
here you don't need the typeof, since you're referring to a property of a defined object. you can write

```js
if (properties['marker-color'] !== undefined) {
```

Instead